itests/org.openhab.core.tests/integration-test.bndrun

```diff
@@ -46,14 +45,14 @@ Provide-Capability: osgi.service;objectClass:List<String>="org.eclipse.smarthome
 	org.eclipse.jetty.server;version='[9.4.11,9.4.12)',\
 	org.eclipse.jetty.servlet;version='[9.4.11,9.4.12)',\
 	org.eclipse.jetty.util;version='[9.4.11,9.4.12)',\
-	org.openhab.core,\
-	org.openhab.core.test,\
-	org.openhab.core.tests,\
 	org.mockito.mockito-core;version='[2.13.0,2.13.1)',\
 	org.objenesis;version='[2.6.0,2.6.1)',\
 	org.osgi.service.event;version='[1.4.0,1.4.1)',\
 	osgi.enroute.junit.wrapper;version='[4.12.0,4.12.1)',\
 	slf4j.api;version='[1.7.21,1.7.22)',\
 	tec.uom.lib.uom-lib-common;version='[1.0.2,1.0.3)',\
 	tec.uom.se;version='[1.0.8,1.0.9)',\
-	osgi.enroute.hamcrest.wrapper;version='[1.3.0,1.3.1)'
+	osgi.enroute.hamcrest.wrapper;version='[1.3.0,1.3.1)',\
+	org.openhab.core;version='[2.5.0,2.5.1)',\
```

Comment:
Will this break after 2.5.1? Why

Reply:
The version range has been chosen by the tooling. So perhaps question about should be discussed upstream (with the tool developers) or by reading their code base:
https://github.com/bndtools/bnd/tree/master/maven/bnd-resolver-maven-plugin